D3D12Bundles/D3D12Bundles.cs

```diff
@@ -60,31 +64,23 @@ struct Vertex {
         // Pipeline objects.
         Viewport mViewport;
         RawRect mScissorRect;
-        IDXGISwapChain3 mSwapChain;
-        ID3D12Device mDevice;
-        readonly ID3D12Resource[] mRenderTargets;
-        ID3D12Resource mDepthStencil;
-        ID3D12CommandAllocator mCommandAllocator;
-        ID3D12CommandQueue mCommandQueue;
+        GraphicsPresenter mPresenter;
+        GraphicsDevice mGraphicsDevice;
         ID3D12RootSignature mRootSignature;
-        ID3D12DescriptorHeap mRtvHeap;
         ID3D12DescriptorHeap mCbvSrvHeap;
-        ID3D12DescriptorHeap mDsvHeap;
         ID3D12DescriptorHeap mSamplerHeap;
-        ID3D12PipelineState mPipelineState1;
-        ID3D12PipelineState mPipelineState2;
-        ID3D12GraphicsCommandList mCommandList;
+        PipelineState mPipelineState1;
+        PipelineState mPipelineState2;
 
         // App resources.
         int mNumIndices;
         ID3D12Resource mVertexBuffer;
         ID3D12Resource mIndexBuffer;
-        ID3D12Resource mTexture;
+        Texture mTexture;
```

Comment:
See individual commit messages on the PR for more details, but basically all of these diffs are replacing the native types with higher level abstractions derived from the DX12GE to ease code maintenance.

There's some more to go, ID3D12RootSignature, and the CBV, SRV, and Sampler heaps, along with models.  But this PR represents a fairly significant chunk of work and it's running fine, so I'm flushing the changes...